Samples/AzureWebSample/OrleansAzureSilos/OrleansConfiguration.xml

```diff
@@ -1,10 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- 
+	This is a sample silo configuration file for use by an Azure worker role acting as an Orleans silo. 
+	The comments illustrate common customizations.
+	Elements and attributes with no comments should not usually need to be modified.
+	For a detailed reference, see "Orleans Configuration Reference.html".
+-->
 <OrleansConfiguration xmlns="urn:orleans">
   <Globals>
     <Liveness LivenessType="AzureTable" />
   </Globals>
 	<Defaults>
-    <Tracing DefaultTraceLevel="Info" TraceToConsole="true" TraceToFile="{0}-{1}.log" >
+	  <!-- 
+	  To turn tracing off, set DefaultTraceLevel="Off" and have no overrides. To see a minimum of messages, set DefaultTraceLevel="Error".
+      For the trace log file name, {0} is the silo name and {1} is the current time. 
+	  Setting WriteTraces to true will cause detailed performance information to be collected and logged about the individual steps in the
+	  message lifecycle. This may be useful debugging performance issues.
+	  -->
+    <Tracing DefaultTraceLevel="Info" TraceToConsole="true" TraceToFile="none" >
```

Comment:
In the past a few people have reported problems with log files not being able to be written to disk when they deploy to Azure Cloud, which prevented roles starting. This was due to the various ways Azure work role VM get constructed having some drives being read-only. Unfortunately it seems to vary significantly for different OS families & versions in Azure. 

The original AzureWebSample just have the WAD trace listener turned on by default and no trace file, but somewhere along the path to here that got changed.

By keeping the TraceToConsole=True the debug output should always be available in Dev Emulator.

I can certainly add tweek the comments in the config files to give clearer explanation of how to turn on trace file for debug. 

I definitely don't think having a default configuration for a sample app that means it won't work out-of-the-box in Azure Cloud is some mainstream deployment scenarios is "better" by any objective scale of measurement ;)